sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_custom_documents.py

```diff
@@ -59,6 +59,9 @@ def print_table(header_names, table_data):
 
 
 def analyze_custom_documents(custom_model_id):
+    # For the Form_1.jpg, it's the test file related to a training data set.
+    # Before analyzing a custom document, should upload the related training dataset into Azure Storage Blob and
```

Comment:
I think it is still better to indicate that the dataset is under the path we will have for DOCUMENTINTELLIGENCE_STORAGE_CONTAINER_SAS_URL

Reply:
for this particular container SAS URL: I am comparing it with our REST API
"AzureBlobSource":
    {
        "prefix": "happyPath/Training",
        "containerUrl": "[<SASURL to the root of this blob storage container>"]
    },
does it mean the URL you provided here will not provide any prefix (subFolder inside the same container)?

Reply:
> I think it is still better to indicate that the dataset is under the path we will have for DOCUMENTINTELLIGENCE_STORAGE_CONTAINER_SAS_URL

Fixed

Reply:
> for this particular container SAS URL: I am comparing it with our REST API "AzureBlobSource": { "prefix": "happyPath/Training", "containerUrl": "["] }, does it mean the URL you provided here will not provide any prefix (subFolder inside the same container)?

Got your point, for Container and Blobs could be accessed with SAS token directly, but folder not. Modified the initialization of AzureBlobContentSource to save trainning set with separate folder instead of Blob Container root, for different type of data.   Thanks for the professional perspective!